module/zfs/zfs_ioctl.c

```diff
@@ -195,6 +195,9 @@
 kmutex_t zfsdev_state_lock;
 zfsdev_state_t *zfsdev_state_list;
 
+/* The bits are stored in decimal format, not octal */
+int zfsdev_umask = 77;
```

Comment:
I felt using a decimal format would be easier for those reading the setting in /sys/module/zfs/parameters/zfsdev_umask, but I am open to switching to octal if people prefer that.